utilities/tools/testclf/testclf.py

```diff
@@ -0,0 +1,162 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright Contributors to the OpenColorIO Project.
+"""
+Test CLF
+========
+
+CLI tool to apply a Common LUT Format (CLF) transform to input RGB data
+to test its accuracy.
+
+This module can also be imported and the function ``test_clf`` called
+to use the tool programmatically.
+"""
```

Comment:
Given that there are CTF files in the repo too, you may want to mention that this tool works with formats other than CLF.  For example:

"In addition to CLF, this tool will work with any other LUT format supported by OpenColorIO, including CTF."

Reply:
Good point. Added the comment.